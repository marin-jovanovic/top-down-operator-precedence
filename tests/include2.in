include "t_include_"
cpp_include "cpp_include_"
