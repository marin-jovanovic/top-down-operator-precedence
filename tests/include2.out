Document
 HeaderManager
  Header
   Include
    "include"
    Literal
     "t_include_"
  HeaderManager
   Header
    CppInclude
     "cpp_include"
     Literal
      "cpp_include_"
   HeaderManager
    end of header instructions
 DefinitionManager
 todo